tests/tests_unit/test_base.py

```diff
@@ -370,6 +370,21 @@ def test_use_method_which_requires_cognite_client__client_not_set(self):
         with pytest.raises(CogniteMissingClientError):
             mr.use()
 
+    @pytest.mark.dsl
+    def test_to_pandas_method(self):
+        import pandas as pd
+
+        from cognite.client.data_classes import Asset
+
+        obj = Asset(id=1)
+
+        result_df = obj.to_pandas()
+
+        expected_df = pd.DataFrame({"value": [1]}, index=["id"])
```

Comment:
If the point of the test was to make sure the ordering of rows stays consistent after your change, you gotta need more rows 😉